src/App.tsx

```diff
@@ -1,16 +1,24 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const onAdd = (movie: Movie) => {
+    setMovies([...movies, movie]);
+  };
```

Comment:
1. naming
2. use callback if you want to access current state
```suggestion
  const handleAddMovie = (movie: Movie) => {
    setMovies((currentMovies) => [...currentMovies, movie]);
  };
```

Reply:
fix it throughout your code